### ISSUE TITLE
attributeDescription should be per-field, not per-validator

### ISSUE THREAD

**Issue by lolmaus** [offirgolan/ember-cp-validations#38]
It is burdensome to pass `attributeDescription` to every validator of a field:

```js
const Validations = buildValidations({
  name: [
    validator('presence', {
      presence: true,
      attributeDescription: 'Course name (language)'
    }),
    validator('length', {
      min: 1,
      attributeDescription: 'Course name (language)'
    }),
    validator('no-whitespace-around', {
      attributeDescription: 'Course name (language)'
    })
  ]
});
``` 

It is obvious that conceptually `attributeDescription` is an entity that belongs to the whole field and should be shared among all validators.

I suggest the following API:

```js
/**
 * Top level method that will ultimately return a mixin with all CP validations
 * @param  {Object} validations  Validation rules
 * @param  {Object} commonOptions  Options to be includd into every validator unless defined explicitly
 * @return {Ember.Mixin}
 */
export default function buildValidations(validations) {
```

which would drastically simplify the above example:

```js
const Validations = buildValidations({
  name: [
    validator('presence', true),
    validator('length', { min: 1 }),
    validator('no-whitespace-around')
  ]
},
{
  attributeDescription: 'Course name (language)'
});
```

**Comment by offirgolan** [comment]
Thanks for the suggestion. I definitely had something like this in my head for future implementation. Ill take a look into this more when I have some time.

**Comment by offirgolan** [comment]
This will be available once we release 2.0! Thanks again for the suggestion :smile:
